client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { overEvery as and } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	LinkQuit,
+	makeTour,
+	Quit,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+import {
+	isAbTestInVariant,
+	isEnabled,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const BoostedPostTour = makeTour(
+	<Tour
+		name="boostedPostSurvey"
+		path="/posts/"
+		version="20170127"
+		when={ and(
+			isEnabled( 'boosted-post-survey' ),
+			isDesktop,
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+		) }
+	>
+		<Step name="init" placement="right">
+			<p>{ translate( 'Interested in increasing your readers?' ) }</p>
```

Comment:
Thanks @dbspringer! In that case, I'd like to propose the following changes -- mostly trims + some rewording for clarity / economy, but also more context + more playfulness, since it wasn't really clear from the existing copy that we're just asking them for their feedback (i.e. a favor), not actually offering a new feature.

popup:
> Wouldn't it be nice if your posts reached a wider audience?

[note: this is assuming the tour starts at /posts, not after or related to any one specific post]

Intro copy:
> We'd like to offer people like you a new way to reach more readers. But we need your help -- answering this short survey will help us create more powerful post-boosting magic. 
>
> Boosting a post promotes it in the “Related Posts” area of WordPress.com sites similar to yours. They look like this: [screenshot]

Survey:

> Q.1: Which of your posts would you like to promote? Copy-and-paste a link to it here:

[note: this step feels superfluous -- would it be possible to pre-populate with the URL of their most recent post? Doesn't this step force them out of the survey?]

> Q.2 How much would you consider spending to boost your post? A bigger budget means more eyes on it.

[note: shouldn't there be "$0" or "I wouldn't pay to boost my post" as an option? At no point up to now was it mentioned that this would be a _paid_ feature.]

> Q.3: Rank which of the following you'd most like to increase (you can drag-and-drop the answers):

Completion screen:

> Thanks for taking the survey.
[The final period was missing. I also wonder if we shouldn't add something along the lines of "Stay tuned for updates!" or something else that feels less abrupt -- I'd be a bit annoyed as a user if I'd just taken a survey and have no idea what the next step is (or even if there is one).

Let me know if I can help with this any further / if you have any questions about my suggestions.

cc @mcsf